docs/src/main/asciidoc/logging.adoc

```diff
@@ -146,6 +146,7 @@ This setting is only evaluated if `includeFormattedMessage` is `true`
 | `includeException` | `false` | Should the stacktrace be included as a own field
 | `serviceContext` | none | Define the Stackdriver service context data (service and version). This allows filtering of error reports for service and version in the https://console.cloud.google.com/errors[Google Cloud Error Reporting View].
 | `customJson` | none | Defines custom json data. Data will be added to the json output.
+| `loggingEventEnhancer` | none | Class names implementing `JsonLoggingEventEnhancer` which modifies the JSON logging output (link:https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/master/spring-cloud-gcp-logging/src/main/java/com/google/cloud/spring/logging/extensions[examples]).
```

Comment:
I'd probably mention that a Logstash tag-enriching implementation is available in the extensions package, just to make it discoverable through search engines.

Reply:
Also, if the tag is repeatable, mention that. It's a repeatable XML tag, right? -- it's not like you can specify multiple classes in the same tag.

Reply:
Done and Done.